crates/subspace-farmer/Cargo.toml

```diff
@@ -69,7 +70,6 @@ thread-priority = "1.1.0"
 tokio = { version = "1.40.0", features = ["macros", "parking_lot", "rt-multi-thread", "signal", "sync", "time"] }
 tokio-stream = { version = "0.1.16", features = ["sync"] }
 tracing = "0.1.40"
-tracing-subscriber = { version = "0.3.18", features = ["env-filter"], optional = true }
```

Comment:
Everywhere you have removed tracing-subscriber, please also remove supports-color.
(Sorry I didn’t catch this earlier!)

Reply:
Done